cudamapper/src/index.cu

```diff
@@ -40,3 +40,135 @@ std::unique_ptr<Index> Index::create_index(std::shared_ptr<DeviceAllocator> allo
 
 } // namespace cudamapper
 } // namespace claragenomics
+
+
+
+namespace claragenomics
+{
+namespace cudamapper
+{
+
+    IndexCache::IndexCache(const Index& index,
+                           const read_id_t first_read_id_in,
```

Comment:
Personal preference: We generally do not use `_in` to indicate parameter direction, `const` should make it clear enough

Reply:
fixed.